.travis.yml

```diff
@@ -7,6 +7,10 @@ go:
     - 1.10.x
     - tip
 
+env:
+    - BUILDAH_ISOLATION=oci
+    - BUILDAH_ISOLATION=chroot
```

Comment:
Does this cause two tests to run, or just override the first ENV with the second?

Reply:
Each test configuration we had becomes two, now.